.github/ISSUE_TEMPLATE/feature-request.md

```diff
@@ -0,0 +1,24 @@
+---
```

Comment:
I think this template, as a whole, needs some more thought put into it.

Reply:
Agreed, I edited it a little bit to be more friendly lol